_posts/2017/2017-07-21-fullstack-london-of-dragons-and-numbers.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "Fullstack London - Of dragons, numbers and IoT"
+date: "2017-07-21 06:34:02"
```

Comment:
publication date: 2017-07-20 > also change in file name then
(if we can handle that tomorrow ;-))